src/main/java/com/example/calender/controller/EmployeeController.java

```diff
@@ -1,46 +1,83 @@
 package com.example.calender.controller;
 
+
+import com.example.calender.dto.dtoEmployee;
 import com.example.calender.entity.Employee;
+import com.example.calender.exception.ResourceNotFoundException;
 import com.example.calender.service.EmployeeService;
+import com.example.calender.service.EmployeeServiceImpl;
+import lombok.Lombok;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @RestController
+@Slf4j()
 @RequestMapping("/employee")
-public class EmployeeController {
+public class EmployeeController{
 
-    public EmployeeService employeeService;
+    @Autowired
+    private ModelMapper modelMapper;
+    @Autowired
+    public EmployeeService employeeServiceImpl;
 
-    public EmployeeController(EmployeeService employeeService){
-        this.employeeService=employeeService;
+    public EmployeeController(EmployeeServiceImpl employeeServiceImpl){
+        this.employeeServiceImpl = employeeServiceImpl;
     }
+
     @GetMapping()
-    List<Employee> getAllEmployees( ){
-      return employeeService.getAllEmployees();
+    List<dtoEmployee> getAllEmployees( ){
+        log.debug("Received Request to display all Employees");
```

Comment:
better keep the logs entirely lower-case, helps in finding the log lines when you don't need to worry about what's upper-case and what isn't